app/lib/nomis_client/base.rb

```diff
@@ -54,6 +55,15 @@ def benchmark_request(path)
 
         response
       end
+
+      def update_json_headers(headers)
+        return unless headers.key?(:Accept) && headers.key?(:'Content-Type')
+
+        {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        }
+      end
```

Comment:
Would this remove existing headers passed into the `post` method? The logic I think I'd expect from this and L14 is:

- Set the `Accept` header unless it has been specified in `params[:headers]`.
- Set the `Content-Type` header unless it has been specified in `params[:headers]`.

That way we'd preserver any explicit headers added by the calling method, but add them otherwise. If that matches your understanding we might want to rename this `add_json_headers`, and merge those with the existing headers rather than overwriting it.